alignak/daemon.py

```diff
@@ -209,8 +209,12 @@ class Daemon(object):
             BoolProp(default=False),
         'daemon_enabled':
             BoolProp(default=True),
-        'spare':
-            BoolProp(default=False),
+        # Todo: spare is not present currently in the daemon.ini file
+        # 'spare':
```

Comment:
daemon cfg file is not used here ! it is the daemon .ini file that is concerned !